### DISCUSSION TITLE
Hidden links and crazy-long HTML generated?

### DISCUSSION THREAD

**Discussion by dogweather** [imfing/hextra#274] [Q&A]
I'm trying to switch from Ananke Theme to Hextra, but I have a weird issue. Every page has a hidden list of links to all the other pages on the site. I have 20,000 pages, so this is a big problem. 

It's not just simple `<a>` tags either: here's a screenshot from the middle of a simple About page. As far as I can tell, I haven't enabled any strange configurations. And what are these line number spans?

<img width="879" alt="Screenshot 2024-02-02 at 11 54 55 PM" src="https://github.com/imfing/hextra/assets/150670/785204ab-6839-41f6-b4ad-45d4d1d214d4">

The page itself doesn't show anything strange. A megabyte of HTML is hidden in there, though:

<img width="1079" alt="Screenshot 2024-02-02 at 11 58 16 PM" src="https://github.com/imfing/hextra/assets/150670/aa040f95-00fa-441e-a284-8e61271e5ac8">

**Comment by imfing** [comment]
Hi looks like it's caused by the mobile navbar that includes every page for the navigation. 

Will provide an option to disable that.

**Reply by dogweather** [reply]
What's the purpose of these empty spans?

**Reply by imfing** [reply]
hmmm, I don't know. I had another look at the screenshot and found that the `<span id="line140*"><span>` looks odd to me.

**Comment by dogweather** [comment]
I'm wondering if I can help fix it. It seems like the problem is: the mobile menu needs to simply contain the current menu and/or sidebar shown. So, it needs to be dynamic to a certain degree. As opposed to currently, where it simply shows the sidebar with links to all pages.

**Reply by imfing** [reply]
you can override it here:

https://github.com/imfing/hextra/blob/7191e259584e339b499a63e2e60cb6d818222e18/layouts/partials/sidebar.html#L25-L29

simply comment out these lines in your own site

**Reply by imfing** [reply]
Would appreciate it if you could help improve the current behavior by providing an option in the config file to enable/disable it.
